### PR TITLE
feat(Log parsing): DOC-7605

### PR DESCRIPTION
**Pull request by barbnewrelic**
Update parsing doc for DOC-7605. **Note to reviewer:** This doc intentionally is long because Logs team wants "one stop shopping" for now. Thanks for understanding!

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by barbnewrelic** [comment]
Note to self: https://github.com/newrelic/docs-website/pull/6058 will probably cause a merge conflict with my PR when I finalize this text

File: src/content/docs/logs/ui-data/parsing.mdx [hunk 1]
```diff
@@ -96,98 +96,175 @@ Here's an overview of how New Relic implements parsing of logs:
...
+  2. Identify the attribute/value pair that acts as a pre-filter for the incoming logs; for example, `entity.name` and the value `Inventory Service`. This pre-filter narrows down the number of logs that need to be processed by your rule, removing unnecessary processing.
+  3. Add the Grok parsing rule; for example:
+
+    ```
+    Inventory error: %{DATA:error_message} for product %{INT:product_id}
+    ```
+
+    Where:
+
+    * `Inventory error` = Your parsing rule's name
```
Comment (urbiz-nr) [inline src/content/docs/logs/ui-data/parsing.mdx:157]: Just a tiny consistency comment, we usually use : instead of = for this.

**Review by urbiz-nr — APPROVED** [review]
Bravo! Left a tiny comment, up to you!

File: src/content/docs/logs/ui-data/parsing.mdx (hunk 1 above)
Comment (barbnewrelic) [reply src/content/docs/logs/ui-data/parsing.mdx:157]: d'oh! You're right! I'll fix. Thanks for catching that!

**Comment by barbnewrelic** [comment]
This has already been reviewed so no need for a second one that autobot created

File: src/content/docs/logs/ui-data/parsing.mdx [hunk 2]
```diff
@@ -96,98 +96,175 @@ Here's an overview of how New Relic implements parsing of logs:
...
+    Where:
+
+    * `Inventory error`: Your parsing rule's name
+    * `error_message`: The error message you want to select
+    * `product_id`: The product ID for Inventory Service
+  
+  4. Test the Grok pattern to see if any incoming logs match.
+  5. Enable and save the parsing rule.
+    
+  Soon you will see that your Inventory Service logs are enhanced with two new fields: `error_message` and `product_id`. From here, you can query on these fields, create charts and dashboards, set alerts, etc.
```
[the remark this answers is not in the document]
Comment (barbnewrelic) [reply src/content/docs/logs/ui-data/parsing.mdx:164]: fixed, thx for catching that!

**Pull request merged**